src/test/java/com/google/api/codegen/testdata/python_doc_types_library.baseline

```diff
@@ -0,0 +1,65 @@
+============== file: google/cloud/proto/example/library_v1/types.py ==============
```

Comment:
> I might be missing the full context of this re-organization. Are the `google.cloud.{gapic,proto}.ex.v1` packages going away and being replaced by these `ex_v1.{gapic,proto}` packages?

They do not have to be necessarily, but it matters that the ultimate names we expose to users are `google.cloud.example_v1` and only things underneath that.

The current implementation in Vision, Speech, NL, etc. has modules that import from the existing namespaces, but changing the namespaces would actually be easier for documentation and such.